specs/opcode/19NOT.md

```diff
@@ -0,0 +1,31 @@
+# NOT opcode
+
+## Procedure
+
+Pop one EVM word `a` from the stack and push output `b` onto the stack.
+
+We prove that `a = NOT b` by showing that the byte-wise relation
+`a[i] XOR b[i] == 255` holds for each of `i = 0..32`.
+
+## Constraints
+
+1. opcodeId checks
+   - opId == OpcodeId(0x19)
+2. state transition:
+   - gc + 2
+   - stack_pointer unchanged
+   - pc + 1
+   - gas + 3
+3. Lookups: 34 busmapping lookups
```

Comment:
Maybe better say: 2 busmapping lookups + 32 fixed table lookups?

Reply:
fixed in 5b9a0d3